export_visual_to_wordpress.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
This feature is very much a work in progress, but I'm gonna keep it here for future's sake. There's no private information in here.

Reply:
With some time apart from this, I think I'll just do it using node as that's the most relevant tech to this stack and it's so much easier to debug.